### ISSUE TITLE
Error when using cyto_plot_gating_scheme() on a GatingHierarchy

### ISSUE THREAD

**Issue by herbstk** [DillonHammill/CytoExploreR#80]
When I use `cyto_plot_gating_scheme()` with a `GatingHierarchy` object I get the error message

 ```
Error in as.list.default(X) : 
  no method for coercing this S4 class to a vector
```
It works without problems when using a `GatingSet` object.
For example I fail to reproduce the example from the help page of `cyto_plot_gating_scheme()` without errors:
```R
# Load in samples
fs <- Activation
gs <- GatingSet(fs)

# Apply compensation
gs <- cyto_compensate(gs)

# Transform fluorescent channels
gs <- cyto_transform(gs)

# Gate using cyto_gate_draw
gt <- Activation_gatingTemplate
gt_gating(gt, gs)

# Gating scheme
cyto_plot_gating_scheme(gs[[4]]) # --> FAILS

cyto_plot_gating_scheme(gs[4]) # --> WORKS
```

Maybe this is related to #71 and will be resolved once the `cyto_plot` family is updated?

**Comment by DillonHammill** [comment]
@herbstk, yes this will be fixed soon. I have just finished making all the updates that I wanted to make to the core of `cyto_plot()` and so I will start working on updating all the wrapper functions today. Hopefully this will all be ready next week. There are a lot of improvements and new features coming to the `cyto_plot()` family and I can't wait to share it with everyone.

**Comment by herbstk** [comment]
Dear @DillonHammill, thank you for your quick response, can't wait for the update. Also, I wanted to say thank you for such a great package, it provides what I was missing so far analysing FCM data with R. Great work!
